README.md

```diff
@@ -19,57 +19,63 @@ remotes::install_github("rstudio/rticles")
 
 The **rticles** package provides a suite of custom [R Markdown](http://rmarkdown.rstudio.com) LaTeX formats and templates for various formats, including:
 
-- [ACM](http://www.acm.org/) articles
+- [ACM : Association for Computing Machinery](https://www.acm.org/publications/about-publications) articles - `acm_article()`
 
-- [ACS](http://pubs.acs.org/) articles
+- [ACS](http://pubs.acs.org/) articles (contributed by @yufree) - `acs_article()`
 
-- [AEA](https://www.aeaweb.org/journals/policies/templates) journal submissions
+- [AEA: American Economic Association](https://www.aeaweb.org/journals/policies/templates) journal submissions (contributed by @sboysel) - `aea_articles()`
 
-- [AGU](https://sites.agu.org/) journal submissions
+- [AGU](https://agupubs.onlinelibrary.wiley.com/) journal submissions (contributed by @eliocamp via #199) - `agu_article()`
 
-- [AMS](https://www.ametsoc.org/) articles
+- [AMS: American Meteorological Society](https://www.ametsoc.org/) articles (contributed by @yufree) - `ams_article()`
 
-- [arXiv](https://arxiv.org/) pre-prints based on George Kour's template (contributed by @alexpghayes via #236)
+- [ASA: American Statistical Association](https://www.amstat.org/) journal submissions - `asa_article()`
 
-- [Biometrics](http://www.biometrics.tibs.org/) articles
+- [arXiv](https://arxiv.org/) pre-prints based on George Kour's template (contributed by @alexpghayes via #236) - `arxiv_article()`
 
-- [Bulletin de l'AMQ](https://www.amq.math.ca/bulletin/) journal submissions
+- [Biometrics](http://www.biometrics.tibs.org/) articles (contributed by @daltonhance via #170) - `biometrics_article()`
 
-- [CTeX](https://ctan.org/pkg/ctex) documents
+- [Bulletin de l'AMQ](https://www.amq.math.ca/bulletin/) journal submissions (contributed by @desautm via #145) - `amq_article()`
 
-- [Elsevier](https://www.elsevier.com) journal submissions
+- [Copernicus Publications](https://publications.copernicus.org) journal submissions (contributed by @nuest via #172) - `copernicus_article()`
 
-- [IEEE Transaction](http://www.ieee.org/publications_standards/publications/authors/author_templates.html) journal submissions
+- [CTeX](https://ctan.org/pkg/ctex) documents - `ctex()`
 
-- [JSS](http://www.jstatsoft.org/) articles
+- [Elsevier](https://www.elsevier.com) journal submissions - `elsevier_article()`
 
-- [JOSS](http://joss.theoj.org/) and [JOSE](https://jose.theoj.org/) articles
+- [Frontiers](https://www.frontiersin.org/) articles (contributed by @zkamvar) - `frontiers_article()`
 
-- [MDPI](http://www.mdpi.com) journal submissions
+- [IEEE Transaction](http://www.ieee.org/publications_standards/publications/authors/author_templates.html) journal submissions (contributed by @Emaasit, @coatless, @espinielli, @nathanweeks , @DunLug) - `ieee_article()`
```

Comment:
IEEE format does not have one main contributor but rather several. I had all the relevant one 
https://github.com/rstudio/rticles/commits/master/inst/rmarkdown/templates/ieee_article

Is this ok ? the list is rather long so I am wondering... 🤔

Reply:
That's ok.